script.py

```diff
@@ -26,7 +26,7 @@ def setUp(usrnm, passwd, cl_id, cl_sc):
 	return reddit
 
 def firstTimeLinux():
-	file_path = '/home/' + os.environ.get('USER') +'/Wallpapers/'
+	file_path = '/home/' + os.environ.get('USER') +'/Pictures/'

```

Comment:
You do not really need to change this, however in a future update I will make this reconfigurable.